index.html

```diff
@@ -4580,6 +4580,319 @@ <h1>Current Issues</h1>
+    needs a persistent, cryptographically verifiable identifier could be
+    assigned a <a>DID</a>, and its <a>DID document</a> could be used as a
+    standard way to retrieve a representation of that schema.
+    </p>
+
+    <h2>Can existing web resources also be assigned DIDs?</h2>
+    <p>
+    Yes, if the controller of a web page or any other web resource wants to
+    assign it a persistent, cryptographically verifiable identifier, the
+    controller can give it a <a>DID</a>. For example, the author of a blog
+    hosted by a blog hosting company (under that hosting company’s own URL)
+    could create a <a>DID</a> for the blog. In the <a>DID document</a>, the
+    author can include an alsoKnownAs property pointing to the current URL of
+    the blog:
+    </p>
+    <code>
+        “alsoKnownAs”: [“https://myblog.blogging-host.example.com/home”]
+    </code>
+    <p>
+    If the author subsequently moves the blog to a different hosting company
+    (or to the author’s own domain), the author can update the <a>DID document</a>
+    to point to the new URL for the blog:
+    </p>
+    <code>
+        “alsoKnownAs”: [“https://myblog.example.com/”]
+    </code>
+    <p>
+    The <a>DID</a> effectively adds a layer of indirection for the blog URL.
+    This layer of indirection is under the control of the author instead of
+    under external administrative authority such as a blog hosting company. This
+    is how <a>DIDs</a> can effectively function as
+    <a href="https://tools.ietf.org/html/rfc8141">URNs (Uniform Resource
+    Names)</a>—persistent identifiers for information resources whose network
+    location may change over time.
+    </p>
+  </section>
+
+  <section class="appendix">
+    <h1>
+    DID Controllers and DID Subjects
+    </h1>
+    <p>
+    To avoid confusion about the relationship between <a>DID controllers</a> and
+    <a>DID subject</a>s, the DID Working Group has found it helpful to classify
+    <a>DID subject</a>s into two disjoint sets based on their relationship to
+    the <a>DID controller</a>.
+    </p>
+    <h2>Set 1: The DID subject <em>is</em> the DID controller</h2>
+    <p>
+    The first case, shown in figure 4, is the common scenario where the
+    <a>DID subject</a> is also the <a>DID controller</a>. This is the case when
+    an individual or organization creates a <a>DID</a> to self-identify.
+    </p>
+        <figure id="controller-subject-equivalence">
+          <img style="margin: auto; display: block; width: 75%;"
+            src="diagrams/figure-b.1-controller-and-subject-equivalence.png" alt="
+            Diagram showing the same graph model as figure 2 except with an
+            equivalence arc from the DID subject to the DID controller.
+          " >
+          <figcaption>
+              The <a>DID subject</a> is the same entity as the <a>DID controller</a>
+          </figcaption>
+        </figure>
+    <p>
```

Comment:
I believe the 'Equivalence' (dotted) arc should be symmetric, ie, with an arrow head on both sides. Note that the `owl:sameAs` is strictly symmetric, and the text refers to it...

Reply:
@iherman Great point, I will fix the version of the diagram I have, and then coordinate the SVGs with you.